alicloud/resource_alicloud_db_instance.go

```diff
@@ -101,6 +101,19 @@ func resourceAlicloudDBInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				ForceNew: true,
 				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					// Unfortunately RDS API doesn't retain the info about the additional vswitch
+					// so after creation, we can only compare the first vswitch to guess if there is a diff
+					return old == Trim(strings.Split(new, COMMA_SEPARATED)[0])
+				},
+			},
+			"vswitch_id_slaves": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				ForceNew: true,
+				Optional: true,
```

Comment:
I think there is no need to add an extra field `vswitch_id_slaves` to support multi vswitch and just input several vswitches into `vswitch_id` with comma.

Reply:
Hi @xiaozhu36,

Thanks for reviewing.

The problem with the multiple vswitches separated with comma is that we run into #2845 
and if we just compute diff on the first vswitch (like the first commit of this PR does), then any change to the additional vswitches are ignored.

Reply:
Hi @xiaozhu36 

Will you have some time soon to review this PR ?

Reply:
Hi @xiaozhu36

I am opened to make any required changes, but still waiting for your feedbacks.